dpnp/dpnp_iface_nanfunctions.py

```diff
@@ -137,6 +137,13 @@ def nanargmax(a, axis=None, out=None, *, keepdims=False):
 
     For full documentation refer to :obj:`numpy.nanargmax`.
 
+    Warning
+    -------
+    This function synchronizes in order to test for all-NaN slices in the array.
+    This may harm performance in some applications. To avoid synchronization,
+    the user is recommended to filter NaNs themselves and use `dpnp.argmax`
+    on the filtered array.
```

Comment:
Here is the warning in the `Returns` ([ref](https://github.com/IntelPython/dpnp/pull/2339/commits/f69ef2821d66d77f9ec74d11cff25a4b2e787232#diff-22fe858223c2cd5661c90febbce48f902c4cb8a2df0d9c0cbd8b0ecc6d23bce2R176)), should it be move here also?